apps/openmw/mwstate/quicksavemanager.cpp

```diff
@@ -1,8 +1,8 @@
 #include "quicksavemanager.hpp"
 
 MWState::QuickSaveManager::QuickSaveManager(std::string &saveName, unsigned int maxSaves)
+  : mSaveName(saveName)
```

Comment:
This is a good change, because the initialiser list is preferable to initialising in the body of the constructor. However this change also seems to be incomplete, since there are several in-body initialisations left.